src/test/logout.test.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import toJson from 'enzyme-to-json';
+import { shallow, render, mount } from 'enzyme';
+import ReactTestUtils from 'react-dom/test-utils';
+import { Logout } from '../components/Logout/Logout';
+
+
+function findLogoutURL() {
+  if (NODE_ENV === 'development') {
```

Comment:
this is unnecessary. REACT_APP_GITHUB_OAUTH_LOGOUT should be set to the correct thing based on environment, by create react app